lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {tagName} - name of the parent to search for.  @example
```

Comment:
this should be `@param {string} tagName - bla bla`. see http://usejsdoc.org/tags-param.html
and put `@example` on its own line